guides/Cloud_development/Heroku.md

```diff
@@ -1,3 +1,19 @@
 ## What is Heroku ?
 
-Heroku is basically a cloud platform for running applications.To deploy an application your code needs to run on a server that will respond to request over the internet but where does Heroku come in ? well Heroku will take your application and make it accesible on the internet with few simple commands.Heroku is a platform service for the developer to upload the code and make your application accesible to everyone, this is called a platform as a servive and is one kind of internet profduct available today along with infrastructure and software.As a service you might see this shortened to paas,laas and saas.What is special about paas product is how simple they make it for you to host your application online.Heroku handles most of the details behind the scenes that are required for your application to run.To deploy your application on Heroku all you need is a few simple command line tools installed and your application code is ready to deploy services like Heroku handles all these background task for you making it much simpler for developers to release a live application.
\ No newline at end of file
+Heroku is basically a cloud platform for running applications. To deploy an application your code needs to run on a server that will respond to request over the internet,but where does Heroku come in ? well Heroku will take your application and make it accesible on the internet with few simple commands.Heroku is a platform service for the developer to upload the code and make your application accesible to everyone, this is called a platform as a servive and is one kind of internet product available today along with infrastructure and software. As a service you might see this shortened to paas,laas and saas.What is special about paas product is that how simple they make it for you to host your application online.Heroku handles most of the details behind the scenes that are required for your application to run. To deploy your application on Heroku all you need is a few simple command line tools installed and your application code is ready to deploy services. Heroku handles all these background task for you making it much simpler for developers to release a live application. The Heroku platform supports development in Ruby on Rails, Java, Node.js, Python, Scala and Clojure.
```

Comment:
1) Please **capitalize**, **W** of well since you're starting a new line.Also, insert comma after it. 
2) Also, **capitalize**, **P** , **S** and **I** of paas, laas and saas since these are services of cloud platform i.e Infrastructure as-a-service goes for IaaS .. et all.